### PR TITLE
Update report-bugs.rst

### PR DESCRIPTION
**Pull request by ben-ba**
Update troubleshooting section to reflect the possiblity to validate the configuration before testing it in the wild.

Driven by; https://github.com/canonical/subiquity/pull/1933

### CODE REVIEW

File: doc/howto/report-bugs.rst [hunk 1]
```diff
@@ -5,6 +5,17 @@ Troubleshooting
...
+Validate your autoinstall 'user-data' configuration against the Autoinstall JSON schema :ref:`JSON schema<autoinstall_JSON_schema>`
+----------------
+
+.. code-block:: bash
+
+    git clone https://github.com/canonical/subiquity
+    cd subiquity && make install_deps
+    ./scripts/validate-autoinstall-user-data.py ../ubuntu-autoinstall/user-data
+
+.. note:: The first line in the user-data file have to be "#cloud-config"
```
Comment (Chris-Peterson444) [inline doc/howto/report-bugs.rst:17]: This isn't necessarily true, but is something we're working towards making more clear in the documentation about when it is.

**Review by Chris-Peterson444 — COMMENTED** [review]
I like this! Although I don't think this should go in the bug reporting section. This is actually something I have been working on and plan to give some more focus in the coming weeks. I would like to have a whole how-to page for Autoinstall validation. See https://github.com/canonical/subiquity/pull/1901 for some of the work so far. Would you mind instead commenting on improvements you would like to see there?

File: doc/howto/report-bugs.rst (hunk 1 above)
Comment (ben-ba) [reply doc/howto/report-bugs.rst:17]: was my experience when testing and when i look and the source code.

**Comment by ben-ba** [comment]
>Would you mind instead commenting on improvements you would like to see there?

Of course. A extra section seems to be the better approach.

**Pull request closed without merging**